resources-domain/resources-service/src/referenceAttributes/referenceAttributeRoutesV0.ts

```diff
@@ -22,9 +22,9 @@ const referenceAttributeRoutes = () => {
   const router: IRouter = Router();
   const { getResourceReferenceAttributeList } = referenceAttributeService();
 
-  router.get('/:list', async (req, res) => {
+  router.get('/*', async (req, res) => {
     const { valueStartsWith, language } = req.query;
-    const { list } = req.params;
+    const list = (req as any).params[0];
```

Comment:
Can we document what's this doing? Is `params` now a list or how is this working? 🤔

Reply:
Yes /* returns a list, which is documented in Express, I'm not sure we should document standard API behaviour